features/step_definitions/advocate_claim_steps.rb

```diff
@@ -1,6 +1,9 @@
-
 include WaitForAjax
 
+Given(/^AGFS reform commenced on "([^"]*)"$/) do |agfs_reform_start_date|
+  allow(Settings).to receive(:agfs_fee_reform_release_date).and_return Date.parse(agfs_reform_start_date)
```

Comment:
Won't put this as a blocker, but I'm not quite sure about this, features specs, IMO should not be mocking internal logic of the application, but let the app to behave as it should naturally. I think this should probably be managed through configuration, not stubbing.

Reply:
such as..